repo/packages/M/marathon/3/config.json

```diff
@@ -0,0 +1,378 @@
+{
+  "additionalProperties" : false,
+  "description" : "Marathon Universe properties",
```

Comment:
I think this should probably be `Marathon service properties`. Universe might not make sense when we eventually re-name the Universe (and certainly doesn't with this package, in the Multiverse).